contracts/bonding/BondingManager.sol

```diff
@@ -71,7 +71,7 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
     mapping (address => Transcoder) private transcoders;
 
     // Keep track of total bonded tokens
```

Comment:
I see that we changed the variable name, but let's also update the comment to specify why this is deprecated.